packs/data/backgrounds.db/raised-by-belief.json

```diff
@@ -5,7 +5,14 @@
     "system": {
         "boosts": {
             "0": {
-                "value": []
+                "value": [
+                    "cha",
+                    "con",
+                    "dex",
+                    "int",
+                    "str",
+                    "wis"
+                ]
```

Comment:
I know the deity items have a boost section, I guess that's not working. We should maybe just fix that? A bit more work but maybe not the hardest thing.